testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/drone/KeycloakDronePostSetup.java

```diff
@@ -102,6 +109,30 @@ private void configureFirefoxDriver(WebDriver driver) {
         }
     }
 
+    private void configureHtmlUnitDriver(WebDriver driver) {
+        if (driver instanceof HtmlUnitDriver) {
+            HtmlUnitDriver htmlUnitDriver = (HtmlUnitDriver) driver;
```

Comment:
Same here, we can avoid type-casting.
```suggestion
        if (driver instanceof HtmlUnitDriver htmlUnitDriver) {
```